modules/twinklespeedy.js

```diff
@@ -1516,6 +1515,15 @@ Twinkle.speedy.callbacks = {
 				}
 			}
 
+			// Scribunto isn't parsed like wikitext, so CSD templates on modules need special handling to work
+			if (params.scribunto) {
```

Comment:
Not super critical, but there's no need for line 2175 (`params.scribunto = mw.config.get('wgPageContentModel') === 'Scribunto';`) to exist anymore, yeah?  This can just be written: 
```suggestion
			if (mw.config.get('wgPageContentModel') === 'Scribunto') {
```